### ISSUE TITLE
Pointer indirections not parsed

### ISSUE THREAD

**Issue by Muzza** [slynch8/10x#65]
See code below
```
struct Type
{
	int i;
};

void main()
{
	Type* a = new Type;
	a->i; // ok

	Type** b = &a;
	b[0]->i;	// ok
	(*b)->i;	// no autocomplete options after typing -> and 'i' will be white
}
```

**Comment by slynch8** [comment]
fixed (0.25.9)

**Comment by Muzza** [comment]
I'm on 25.12 and this still does not work for me.

**Comment by slynch8** [comment]
ah, I fixed the goto-def but not the autocomplete, I'll get that fixed...

**Comment by slynch8** [comment]
fixed (0.25.13)
